pkg/datapath/loader/loader.go

```diff
@@ -132,6 +138,68 @@ func nullifyStringSubstitutions(strings map[string]string) map[string]string {
 	return nullStrings
 }
 
+// patchHostNetdevHeaderFile creates a header file corresponding to each new
+// device to which bpf_host is attached, including the second host device and
+// native devices. The goal is to simplify inspection of Cilium state and avoid
+// having to read symbols manually from the object files.
+func patchHostNetdevHeaderFile(stateDir, ifName string, mac mac.MAC, ifIndex uint32,
+	secctxFromIpcache uint32, nodePort bool) (string, error) {
+	netdevHeaderFilename := strings.Replace(common.CHeaderFileName, ".h", "_"+ifName+".h", 1)
+	netdevHeaderFile := path.Join(stateDir, netdevHeaderFilename)
+	headerFile := path.Join(stateDir, common.CHeaderFileName)
```

Comment:
The use of "netdev header" here had me initially assuming this is all referring to `netdev_config.h`, but it doesn't, this is all `ep_config.h` (or `ep_config_<IFNAME>.h`). We might want to re-evaluate whether the `netdev_config.h` is even necessary any more and/or consider using a different name through these paths just to make sure we don't lead anyone else down this confusing rabbit hole :-) .